### ISSUE TITLE
In the games tab, "Browse..." doesn't work

### ISSUE THREAD

**Issue by unixatch** [hrydgard/ppsspp#19778]
### Game or games this happens in

None

### What area of the game / PPSSPP

None

### What should happen

It should have access to the games folder without issues but instead there's this:
![Screenshot_2024-12-29-10-31-34-98_2f85358b2198d26f8aca533d68bee793](https://github.com/user-attachments/assets/d71ebdef-8148-404f-ad0c-6901081f5e6d)

### Platform

Android

### Mobile device model or graphics card (GPU)

Realme 7 5G, RMX2111

### PPSSPP version affected

v1.18.1-g70cdde

### Last working version

v1.17

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://www.ppsspp.org/devbuilds) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [ ] Try without any cheats and without loading any save states.
- [X] Include logs or screenshots of issue.

**Comment by sum2012** [comment]
@unixatch Try AndroidLegacy build in https://www.ppsspp.org/devbuilds/

**Comment by unixatch** [comment]
> [@unixatch](https://github.com/unixatch) Try AndroidLegacy build in https://www.ppsspp.org/devbuilds/

Yes, it does work with the legacy build but not with the normal build

**Comment by hrydgard** [comment]
This does work on most devices so I don't really know what the problem might be here :/

**Comment by sum2012** [comment]
Maybe it used custom android system build without some feature
